getting-started.md

```diff
@@ -163,3 +163,19 @@ Run:
 ```bash
 curl -v -H "X-Debug: true" https://<your_domain>/
 ```
+
+### (Optional) Use environment variables
+
+The authentication parameters used by the `hlx deploy` and `hlx publish` tasks can also be specified as environment variables. You can put this variables into an `.env` file which will be automatically picked up the the client.
```

Comment:
I would write:

All command line arguments can also be provided as _environment variables_, by prefixing them with `HLX_`. For example the `--log-level` can be set via the `HLX_LOG_LEVEL` variable; the `--wsk-namespace` via `HLX_WSK_NAMESPACE` etc. You can put those variables into an `.env` file which will be automatically picked up by the cli.